src/Numerics/DGMethods/balance_law_interface.jl

```diff
@@ -305,3 +305,9 @@ num_hyperdiffusive(m::BalanceLaw, FT) = varsize(vars_hyperdiffusive(m, FT))
 num_integrals(m::BalanceLaw, FT) = varsize(vars_integrals(m, FT))
 num_reverse_integrals(m::BalanceLaw, FT) =
     varsize(vars_reverse_integrals(m, FT))
+
+### split explicit functions
+function initialize_states! end
+function tendency_from_slow_to_fast! end
+function cummulate_fast_solution! end
+function reconcile_from_fast_to_slow! end
```

Comment:
Probably worth documenting what these functions are supposed to do. What the arguments are, and what the expected behavior is.

Reply:
yep, I'll probably save that for the next PR where I define them.